kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVProperties.java

```diff
@@ -35,6 +35,7 @@
   public static final String HEADER_PROPERTY = "kite.csv.header";
   public static final String HAS_HEADER_PROPERTY = "kite.csv.has-header";
   public static final String LINES_TO_SKIP_PROPERTY = "kite.csv.lines-to-skip";
+  public static final String LINES_TO_SAMPLE_PROPERTY = "kite.csv.lines-to-sample";
```

Comment:
I don't think this is something that needs to be in the CSVProperties. Those are for describing how you would the contents of a CSV file and sampling isn't something you need to know to decode the CSV. Just passing the number of lines to use like we do in [JsonUtil](https://github.com/kite-sdk/kite/blob/master/kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/JsonUtil.java#L517) should be sufficient.